### PR TITLE
FINERACT-822  Update and add more checks to be enabled

### PR DESCRIPTION
**Pull request by percyashu**
FINERACT-822

### CODE REVIEW

**Comment by percyashu** [comment]
From https://github.com/apache/fineract/pull/971

**Comment by vorburger** [comment]
I expect this will fail to build, as per #971. 

Just do only the adding of the checks here, not don't upgrade, in this PR (or a new one). 

Adjust title and commit message.

**Comment by percyashu** [comment]
@vorburger Will do only for the checks

**Review by vorburger — APPROVED** [review]
LGTM (if it passes the build, obviously).

**Comment by percyashu** [comment]
@vorburger am sure this will fail since the checks I added aren't available for ver 2.3.4

**Comment by percyashu** [comment]
@vorburger should I update it manually

File: fineract-provider/build.gradle
Comment (xurror) [inline fineract-provider/build.gradle:359]: Seems this is not a valid check, maybe a typo or maybe you meant something else and wrote something else. I can merge this once this is resolved.

**Comment by percyashu** [comment]
Since this passes, #971 can be closed

**Review by xurror — COMMENTED** [review]
I'll just merge this now. I don't see why we need to hold this on longer.

**Pull request merged**
